handler.go

```diff
@@ -49,8 +52,11 @@ func (h *Handler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 	case path == "/services/":
 		h.serveListServices(res, req)
 
+	case strings.HasPrefix(path, "/pods"):
+		h.serveListPods(res, req)
```

Comment:
Is your intention to keep this model permanently? If not then leaving a comment explaining the rough plan you have in mind would help whomever will be taking care of finalizing this work later on.

Reply:
done.